docs/tasks/sync_broker.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Sync a Broker
+layout: docwithnav
+---
+
+A broker's catalog may occasionaly change the services it offers. A broker
```

Comment:
nits about the wording here: a broker doesn't resync - - svcat resyncs.   

like everywhere else, spell out sync.  Good details though Jonathan.